### ISSUE TITLE
Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyrax#7108]
When checking against WCAG 2.2 AA, the color contrast for the "User Collection" type is not high enough.
Tested on f6.nurax.samvera.org

![Image](https://github.com/user-attachments/assets/fc423e1e-c2ab-4784-90c7-964eaff526fb)
